javascript-methods/main.js

```diff
@@ -0,0 +1,73 @@
+const numberOne = 7;
+const numberTwo = 10;
+const numberThree = 35;
+const maximumValue = Math.max(numberOne, numberTwo, numberThree);
+
+console.log('maximumValue:', maximumValue);
+
+const heroes = ['Superman', 'Batman', 'Robin', 'Green Arrow'];
+
+let randomNumber = Math.random(heroes);
```

Comment:
`Math.random()` does not expect any arguments. So you shouldn't be passing in anything when you call it